lib/screens/profile_screens/contribution/contribution_detail/contribution_detail_screen.dart

```diff
@@ -1,25 +1,79 @@
 import 'package:flutter/material.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/domain-shared/ui_constants.dart';
+import 'package:seeds/screens/profile_screens/contribution/contribution_detail/components/contribution_detail_subtitle.dart';
 import 'package:seeds/screens/profile_screens/contribution/interactor/viewmodels/page_commands.dart';
+import 'package:step_progress_indicator/step_progress_indicator.dart';
 
 class ContributionDetailScreen extends StatelessWidget {
   const ContributionDetailScreen({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     final scoreDetails = ModalRoute.of(context)?.settings.arguments as NavigateToScoreDetails?;
-
+    final double width = MediaQuery.of(context).size.width;
     return Scaffold(
-      appBar: AppBar(),
-      body: SafeArea(
-        child: Center(
-          child: Column(
-            children: [
-              Text(scoreDetails?.scoreType ?? ''),
-              Text(scoreDetails?.score.toString() ?? ''),
-              Text(scoreDetails?.title ?? ''),
-              Text(scoreDetails?.subtitle ?? ''),
-            ],
+      appBar: AppBar(
+          title: Row(
+        children: [
+          Text(scoreDetails?.scoreType ?? ""),
+          const Text(' Score'),
+        ],
+      )),
+      bottomSheet: Row(
+        crossAxisAlignment: CrossAxisAlignment.end,
+        children: [
+          Image.asset(
+            'assets/images/contribution/contribution_detail/petals_left.png',
+            width: width * 0.46,
+            fit: BoxFit.fitWidth,
           ),
+          const Expanded(child: SizedBox.shrink()),
```

Comment:
Trick advice: there is a widget called `Spacer()` that does the same.

Reply:
changed to spacer. Thanks for the advice